src/client/components/VoyageDetailsContainer.js

```diff
@@ -1,5 +1,180 @@
-import React from "react";
+import React, { Component } from "react";
+import axios from "axios";
+import VoyageDetails from "./VoyageDetails";
+import CostWidget from "./CostWidget";
+import SuggestedRouteTable from "./SuggestedRouteTable";
 
-export default function VoyageDetailsContainer({ match }) {
-  return <div>This page is about the Voyage ID = {match.params.voyage_id}</div>;
+export default class VoyageDetailsContainer extends Component {
+  state = {
+    voyageId: this.props.match.params.voyage_id,
+    vessel_report: [],
+    suggested_routes: [],
+    vessel_report_id: [],
+    suggested_route_id: [],
+    voyageList: null,
+    selected_route_id: null,
+    selected_route_table: [],
+    departure_time: "",
+    hfocosts: null,
+    lsfocosts: null,
+    vessel_id: null,
+    vesselName: "",
+    departsFrom: "",
+    arrivesAt: "",
+    departure_time: "",
+    vessel_eta: "",
+    date: "",
+    latitude: null,
+    longitude: null,
+    hsfo: null,
+    ulsfo: null,
+    hfo: null,
+    lsfo: null,
+    hfocosts: null,
+    lsfocosts: null,
+    totalcost: null,
+    depart_from_port: null
+    //arrive_at_port: null
+  };
+
+  componentDidMount() {
+    const headerObject = {
+      // method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        authorization:
+          "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwicm9sZSI6InN1cGVydXNlciIsImlhdCI6MTU3MTM4MTI0NSwiZXhwIjoxNTcxNDE3MjQ1fQ.AWtUrRse7Q2_Z0FC0zcAqK1r2BR8BVBCaUz4Snc3Iy0"
```

Comment:
Nice that you've set the header once for this file. Make sure to get the token from here, as soon as this PR is merged: https://github.com/HackYourFuture-CPH/ov-class09-fp/pull/137